consensus/src/simplex/actors/batcher/actor.rs

```diff
@@ -28,6 +29,17 @@ use rand::{CryptoRng, Rng};
 use std::{collections::BTreeMap, sync::Arc};
 use tracing::{debug, trace, warn};
 
+/// Result of adding a vote to a round.
+enum AddResult<D: Digest> {
+    /// Vote was not added (duplicate, invalid, or certificate already exists).
+    Skipped,
+    /// Vote was added and needs verification.
+    Added,
+    /// Vote was added and this is the leader's first notarize vote (forward proposal immediately).
+    LeaderProposal(Proposal<D>),
+}
+
+/// Per-view state for vote accumulation and certificate tracking.
```

Comment:
add comment about how we don't care/think about equivocation here (where leader proposal differs from cert -> this is handled by voter)